setup/README.md

```diff
@@ -18,3 +20,6 @@ Add execution permission:
 
 Run:
 ``` sudo ./setup.sh ```
+
+Or with one-command run:
+wget -O - https://raw.githubusercontent.com/netgroup-polito/edge-infrastructure-ansible/main/setup/edge-pc-local-setup.sh | sudo bash
```

Comment:
I have added this command to the README.md. 
I believe it could be useful, please let me know if you think it is not necessary and I will remove it.